lib/frontend/github-oauth.js

```diff
@@ -34,27 +31,26 @@ module.exports = function (opts) {
     });
   }
 
-  function login(req, resp) {
-    const u = `https://${host}/login/oauth/authorize` +
-        `?client_id=${opts.githubClient
-        }${opts.scope ? `&scope=${opts.scope}` : ''
-        }&redirect_uri=${redirectURI
-        }&state=${state}`;
-
-    resp.statusCode = 302;
-    resp.setHeader('location', u);
-    resp.end();
+  function login(req, res, redirectUrl) {
+    // TODO: We really should be using an Auth library for this, like @octokit/github-auth
```

Comment:
Do we have a follow-up ticket for this so this doesn't get lost in the tech debt void?

Reply:
Yep, it's in the wishlist :)